policyengine_us/variables/gov/states/ny/tax/income/credits/ctc/ny_ctc_post_2024_eligible.py

```diff
@@ -0,0 +1,42 @@
+from policyengine_us.model_api import *
+
+
+class ny_ctc_post_2024_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "NY CTC post-2024 eligibility"
+    documentation = (
+        "Whether the tax unit is eligible for NY CTC under post-2024 rules"
+    )
+    definition_period = YEAR
+    reference = "https://www.nysenate.gov/legislation/laws/TAX/606"  # (c-1)
+    defined_for = StateCode.NY
+
+    def formula(tax_unit, period, parameters):
```

Comment:
i forgot we have this functionality which seems perfect for this
```suggestion
    def formula_2024(tax_unit, period, parameters):
```

Reply:
Confirmed for now we will avoid `formula_yyyy` because it prevents parametric reforms to switch structures